_includes/head.html

```diff
@@ -53,11 +53,13 @@
 
     ga('create', '{{site.googleanalytics.account}}', 'codurance.com');
     ga('send', 'pageview');
+    window.googleAnalyticsEvents = { environment: {ga:ga}, functions: []};
```

Comment:
@alvarogarcia7 Should all GA reporting move to this events model in future? (I don't have any experience implementing GA on a website, no idea what best practices are)